src/client/js/Panels/PropertyEditor/PropertyEditorPanelController.js

```diff
@@ -66,19 +66,10 @@ define(['js/logger',
             self._logger.debug('activeSelection changed', activeSelection);
             var activeNodeId = WebGMEGlobal.State.getActiveObject();
             if (activeSelection && activeSelection.length > 0) {
-                if (activeSelection.length === 1) {
-                    //https://github.com/webgme/webgme/issues/326
-                    if (self.startsWith(activeSelection[0], activeNodeId)) {
```

Comment:
yes. I've checked the different scenarios and it seems to be consistent among them (that is why I removed the 'if nothing selected, let's select the activeNode part' from other UI parts). The original issue was coming from the confusion between the active selection and active node.
the only thing is that now if there is no selected item, the activeNode will be shown in the property editor, which can be a bit ambiguous in a split view, but otherwise clear in my opinion.